src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -93,13 +94,14 @@ protected virtual IEnumerable<FieldType> ProvideFields()
         /// <see cref="GetArgumentInformation{TParameterType}(FieldType, ParameterInfo)">GetArgumentInformation</see>, building
         /// a list of query arguments and expressions as necessary. Then a field resolver is built around the method.
         /// </summary>
-        protected void BuildFieldType(FieldType fieldType, MemberInfo memberInfo)
+        protected void BuildFieldType(EventStreamFieldType fieldType, MemberInfo memberInfo)
         {
             if (memberInfo is PropertyInfo propertyInfo)
             {
                 var resolver = new MemberResolver(propertyInfo, BuildMemberInstanceExpression(memberInfo));
                 fieldType.Arguments = null;
                 fieldType.Resolver = resolver;
+                fieldType.AsyncSubscriber = null;
```

Comment:
Is it necessary?

Reply:
Not necessarily, but the xml comment says specifically that it overwrites the values in `Arguments`, `Resolver` and `AsyncSubscriber`.  So that's what I coded it to do.